editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-component-renderer.tsx

```diff
@@ -69,12 +70,34 @@ export function createComponentRendererComponent(params: {
   filePath: string
   mutableContextRef: React.MutableRefObject<MutableUtopiaCtxRefData>
 }): ComponentRendererComponent {
-  const Component = (realPassedPropsIncludingUtopiaSpecialStuff: any) => {
+  function Component() {
+    // eslint-disable-next-line prefer-rest-params
```

Comment:
why not go with a rest param on Component() ?

Reply:
Originally because I was testing things out, but I don't think there should be any reason why it can't be like that now.

Reply:
Made this change in 1645db31b.